cyhy_report/customer/report.mustache

```diff
@@ -590,26 +593,26 @@ As you review the report, you may have additional questions. Check out the answe
         {\RCSourceSansProTwelve <<#calc.address_count_up_flag>>Increase of \textbf{\numprint{<<&calc.address_count_up>>}}<</calc.address_count_up_flag>>
         <<#calc.address_count_down_flag>>Decrease of \textbf{\numprint{<<&calc.address_count_down>>}}<</calc.address_count_down_flag>>
         <<#calc.address_count_flat_flag>>No Change<</calc.address_count_flat_flag>>}
-      \end{minipage}  % END: ADDRESSES OWNED
+      \end{minipage}  % END: ASSETS OWNED
       \begin{minipage}[t]{0.1\textwidth}
          \  % Spacer column
       \end{minipage}
       \begin{minipage}[t]{0.45\textwidth}
-        \textcolor{rc-dark-green}{\RCSourceSansProBoldFourteen ADDRESSES SCANNED}
+        \textcolor{rc-dark-green}{\RCSourceSansProBoldFourteen LAST SCOPE UPDATE}
 
         \vspace*{-1.5mm}  % blank line needed on previous line for vertical spacing
         \textcolor{rc-dark-green}{\rule{\textwidth}{0.4mm}}
 
         \vspace*{1mm}  % blank line needed on previous line for vertical spacing
-        {\RCSourceSansProSemiBoldEighteen \numprint{<<&calc.addresses_scanned>>}} \scalegraphicsnofig[1.2]{assets/<<#calc.addresses_scanned_up_flag>>grey-up-arrow<</calc.addresses_scanned_up_flag>><<#calc.addresses_scanned_down_flag>>grey-down-arrow<</calc.addresses_scanned_down_flag>><<#calc.addresses_scanned_flat_flag>>no-change-blue-arrows<</calc.addresses_scanned_flat_flag>>}
+        %{\RCSourceSansProSemiBoldEighteen \numprint{<<&calc.addresses_scanned>>}} \scalegraphicsnofig[1.2]{assets/<<#calc.addresses_scanned_up_flag>>grey-up-arrow<</calc.addresses_scanned_up_flag>><<#calc.addresses_scanned_down_flag>>grey-down-arrow<</calc.addresses_scanned_down_flag>><<#calc.addresses_scanned_flat_flag>>no-change-blue-arrows<</calc.addresses_scanned_flat_flag>>}
 
-        {\RCSourceSansProTwelve <<#calc.addresses_scanned_up_flag>>Increase of \textbf{\numprint{<<&calc.addresses_scanned_up>>}}<</calc.addresses_scanned_up_flag>>
-        <<#calc.addresses_scanned_down_flag>>Decrease of \textbf{\numprint{<<&calc.addresses_scanned_down>>}}<</calc.addresses_scanned_down_flag>>
-        <<#calc.addresses_scanned_flat_flag>>No Change<</calc.addresses_scanned_flat_flag>>
+        %{\RCSourceSansProTwelve <<#calc.addresses_scanned_up_flag>>Increase of \textbf{\numprint{<<&calc.addresses_scanned_up>>}}<</calc.addresses_scanned_up_flag>>
+        %<<#calc.addresses_scanned_down_flag>>Decrease of \textbf{\numprint{<<&calc.addresses_scanned_down>>}}<</calc.addresses_scanned_down_flag>>
+        %<<#calc.addresses_scanned_flat_flag>>No Change<</calc.addresses_scanned_flat_flag>>
 
         % blank line needed on previous line for vertical spacing
-        \numprint{<<&calc.addresses_scanned_percent>>}\% of addresses scanned}
-      \end{minipage}  % END: ADDRESSES SCANNED
+        %\numprint{<<&calc.addresses_scanned_percent>>}\% of addresses scanned}
```

Comment:
Can we remove this line versus commenting it out?
```suggestion
```

Reply:
The value of the `calc.addresses_scanned_percent` number is for new stakeholders (especially those with a large address space), so that they understand how much of their scope we have scanned so far.  Without it, they may be misled into thinking that we have fully scanned their address space, when we have not.

If you do want to fully remove the `calc.addresses_scanned_percent`, you should also remove the corresponding code that generates this value and stores it in the JSON file used to render the report.  However, if you still want to provide this metric in the CSV attachments, then don't remove the code that generates it.

Reply:
Please see commit a945299.

Reply:
That is good.

Reply:
@dav3r - I am fine with resolving this conversation, but I wanted to give you a chance to ensure that your comment was addressed.  Therefore, please feel free to resolve this conversation when and if you feet it apporpriate.

Reply:
I spoke with @jeffkause earlier today and we are on the same page, so I'm going to resolve this conversation.